/**:
    ros__parameters:
        device_port: "/dev/ttyUSB0"
        baud_rate: 1000000
        protocol_version: 2.0
        ids: [0, 1, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21]
        dynamixels.max_position_limits: [3.14159, 3.14159, 3.14159, 3.14159, 3.14159, 3.14159, 3.14159, 3.14159, 3.14159, 3.14159, 3.14159, 3.14159, 3.14159, 3.14159, 3.14159, 3.14159, 3.14159, 3.14159, 3.14159]
        dynamixels.min_position_limits: [-3.14159, -3.14159, -3.14159, -3.14159, -3.14159, -3.14159, -3.14159, -3.14159, -3.14159, -3.14159, -3.14159, -3.14159, -3.14159, -3.14159, -3.14159, -3.14159, -3.14159, -3.14159, -3.14159]
        dynamixels.max_velocity_limits: [210, 210, 210, 210, 210, 210, 210, 210, 210, 210, 210, 210, 210, 210, 210, 210, 210, 210, 210]
        dynamixels.temperature_limits: [80, 80, 80, 80, 80, 80, 80, 80, 80, 80, 80, 80, 80, 80, 80, 80, 80, 80, 80]
